includes/class-functions.php

```diff
@@ -251,18 +251,40 @@ public function get_affiliate_email() {
 	}
 
 	/**
-	* Get the affiliate's gravatar
+	* Gets the affiliate's gravatar
 	*
 	* @since 1.0.0
+	*
+	* @param array $atts Attributes passed from the shortcode.
+	* @return string Image tag with src to gravatar.
 	*/
-   public function get_affiliate_gravatar() {
+   public function get_affiliate_gravatar( $atts ) {
 
 	   $affiliate_id = $this->get_affiliate_id();
 
 	   if ( $affiliate_id ) {
 
+			$atts = shortcode_atts(
+				array(
+					'size' => 96,
+				), $atts, 'affiliate_info_gravatar'
+			);
+			/**
+			 * Filters the gravatar defaults.
+			 *
+			 * @since 1.0.0
+			 *
+			 * @param array $args {
+			 *
+			 *     @type int    $size    Avatar display size.
+			 *     @type string $default Absolute location of the default avatar to use (used when the
+			 *                           person has no email address associated with them). Left blank
+			 *                           will render placeholder image.
+			 *     @type string $alt     Alt text. Defaults to affiliate name.
```

Comment:
```suggestion
			 *     Arguments for retrieving the affiliate avatar.
			 *
			 *     @type int    $size    Avatar display size.
			 *     @type string $default Absolute location of the default avatar to use (used when the
			 *                           person has no email address associated with them). Left blank
			 *                           will render placeholder image.
			 *     @type string $alt     Alt text. Defaults to affiliate name.
```
Add a description for the `$args` hash notation.